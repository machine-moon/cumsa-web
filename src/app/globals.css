@import "tailwindcss";

/* ========== CSS Variables ========== */
:root {
  --background: #edfaff;
  --foreground: #0f172a;
  --surface: #ffffff;
  --muted: #64748b;
  --navy: #2a7487;
  --blue: #5aced8;
  --red: #c73221;
  --green: #169845ff;
  --header-top: #123a66;
}

/* ========== Base Styles ========== */
body {
  background: var(--background);
  color: var(--foreground);
  font-family: Arial, Helvetica, sans-serif;
}

a {
  text-underline-offset: 4px;
}

/* ========== Links ========== */
.link-cta {
  color: var(--blue);
  font-weight: 700;
  text-decoration: underline;
  text-underline-offset: 3px;
  transition: color 150ms ease;
}

.link-cta:hover {
  color: var(--red);
}

/* ========== Buttons ========== */
.btn-cozy {
  display: inline-flex;
  align-items: center;
  justify-content: center;
  padding: 0.5rem 1rem;
  border-radius: 0.6rem;
  font-weight: 600;
  font-size: 0.9rem;
  transition:
    transform 150ms ease,
    box-shadow 150ms ease,
    opacity 150ms ease;
}

.btn-cozy:hover {
  transform: translateY(-1px);
  box-shadow: 0 6px 16px rgba(0, 0, 0, 0.08);
}

.btn-primary {
  background: var(--blue);
  color: white;
}

.btn-outline {
  background: var(--surface);
  color: var(--foreground);
  border: 1px solid rgba(0, 0, 0, 0.12);
}

/* ========== Accessibility ========== */
:focus-visible {
  outline: 2px solid color-mix(in oklab, var(--blue) 70%, white);
  outline-offset: 2px;
}

/* ========== Layout Containers ========== */
.container-base {
  margin-left: auto;
  margin-right: auto;
  max-width: 80rem;
  padding-left: 1rem;
  padding-right: 1rem;
}

@media (min-width: 640px) {
  .container-base {
    padding-left: 1.5rem;
    padding-right: 1.5rem;
  }
}

@media (min-width: 1024px) {
  .container-base {
    padding-left: 2rem;
    padding-right: 2rem;
  }
}

/* ========== Transition Utility ========== */
.transition-base {
  transition: all 200ms ease-out;
}

/* ========== Cards ========== */
.card {
  background: var(--surface);
  border: 1px solid rgba(0, 0, 0, 0.12);
  border-radius: 0.75rem;
}

/* ========== Typography ========== */
.title {
  color: var(--red);
}

.typewriter-cursor {
  color: var(--blue);
}

/* ========== Icon and Logo ========== */
.tint-blue {
  filter: invert(28%) sepia(88%) saturate(3612%) hue-rotate(211deg) brightness(93%) contrast(98%);
}

.logo-mask-blue {
  background-color: var(--blue);
  -webkit-mask-image: url("/CU-MSA-LOGO-blank.png");
  mask-image: url("/CU-MSA-LOGO-blank.png");
  -webkit-mask-repeat: no-repeat;
  mask-repeat: no-repeat;
  -webkit-mask-position: center;
  mask-position: center;
  -webkit-mask-size: contain;
  mask-size: contain;
  display: inline-block;
}

/* ========== Special Surfaces ========== */
.portal-glow {
  width: 100%;
  max-width: 72rem;
  border-radius: 0.75rem;
  box-shadow:
    0 4px 32px 0 #5aced880,
    0 0 0 2px #5aced850;
  background: #fff;
  border: none;
  padding: 0;
}

.frost-card {
  background: color-mix(in oklab, var(--header-top) 45%, transparent);
  border: 1px solid rgba(255, 255, 255, 0.18);
  border-radius: 1rem;
  box-shadow:
    0 20px 70px rgba(10, 22, 40, 0.25),
    0 2px 6px rgba(0, 0, 0, 0.05);
  backdrop-filter: blur(10px) saturate(120%);
}

/* ========== Hero Section ========== */
.hero-overlay {
  background:
    radial-gradient(1000px 500px at 20% 20%, rgba(90, 206, 216, 0.18), transparent 60%),
    radial-gradient(800px 420px at 80% 30%, rgba(199, 50, 33, 0.1), transparent 60%),
    linear-gradient(to bottom, rgba(18, 58, 102, 0.25), rgba(18, 58, 102, 0.5));
}

.hero-glow {
  position: absolute;
  pointer-events: none;
  filter: blur(48px);
  opacity: 0.5;
}

.hero-glow-1 {
  width: 40vw;
  height: 40vh;
  left: -10vw;
  top: 20vh;
  background: radial-gradient(circle at center, rgba(90, 206, 216, 0.28), transparent 60%);
}

.hero-glow-2 {
  width: 35vw;
  height: 35vh;
  right: -8vw;
  bottom: 10vh;
  background: radial-gradient(circle at center, rgba(199, 50, 33, 0.14), transparent 60%);
}

@media (max-width: 640px) {
  .hero-glow {
    display: none;
  }
}

/* ========== Animations ========== */
/* Fade / Slide / Spin / Float / Sparkle */
@keyframes fade-in {
  from {
    opacity: 0;
    transform: translateY(20px);
  }

  to {
    opacity: 1;
    transform: translateY(0);
  }
}

@keyframes slide-in-left {
  from {
    opacity: 0;
    transform: translateX(-30px);
  }

  to {
    opacity: 1;
    transform: translateX(0);
  }
}

@keyframes slide-in-right {
  from {
    opacity: 0;
    transform: translateX(30px);
  }

  to {
    opacity: 1;
    transform: translateX(0);
  }
}

@keyframes spin {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
}

@keyframes float {
  0%,
  100% {
    transform: translateY(0);
  }

  50% {
    transform: translateY(-10px);
  }
}

@keyframes sparkle {
  0%,
  100% {
    opacity: 0;
    transform: scale(0.5);
  }

  50% {
    opacity: 1;
    transform: scale(1);
  }
}

.animate-fade-in {
  animation: fade-in 0.6s ease-out;
}

.animate-slide-in-left {
  animation: slide-in-left 0.6s ease-out 0.2s both;
}

.animate-slide-in-right {
  animation: slide-in-right 0.6s ease-out 0.4s both;
}

.animate-spin {
  animation: spin 1s linear infinite;
}

.animate-float {
  animation: float 3s ease-in-out infinite;
}

.animate-sparkle {
  animation: sparkle 2s ease-in-out infinite;
}

/* ========== Custom Scrollbar ========== */
.custom-scroll {
  scrollbar-width: thin;
  scrollbar-color: var(--blue) transparent;
}

.custom-scroll::-webkit-scrollbar {
  width: 8px;
}

.custom-scroll::-webkit-scrollbar-track {
  background: transparent;
}

.custom-scroll::-webkit-scrollbar-thumb {
  background: color-mix(in oklab, var(--blue) 75%, white);
  border-radius: 8px;
  border: 2px solid transparent;
  background-clip: content-box;
}

.custom-scroll::-webkit-scrollbar-thumb:hover {
  background: var(--blue);
}
