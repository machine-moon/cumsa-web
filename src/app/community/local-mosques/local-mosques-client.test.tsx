import { test, expect } from "@jest/globals";
import { render, act } from "@testing-library/react";
import LocalMosquesClient from "./local-mosques-client";

test("local mosques client snapshot", async () => {
  let component: ReturnType<typeof render> | undefined;

  await act(async () => {
    component = render(<LocalMosquesClient />);
  });

  if (!component) {
    throw new Error("Component failed to render");
  }

  expect(component.asFragment()).toMatchSnapshot();
});
