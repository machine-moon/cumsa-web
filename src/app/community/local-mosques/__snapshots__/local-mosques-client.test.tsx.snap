// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`local mosques client snapshot 1`] = `
<DocumentFragment>
  <div
    class="mt-8 grid gap-8 md:grid-cols-2"
  >
    <div
      class="space-y-3"
    >
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="true"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Ottawa Mosque (OMA)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform rotate-180"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
        <div
          class="px-4 pb-4 text-sm text-gray-700 space-y-1"
        >
          <div
            class="font-semibold text-[var(--red)]"
          >
            Information
          </div>
          <div>
            251 Northwestern Ave, Ottawa, ON
          </div>
          <div>
            (613) 722-8763
          </div>
          <div>
            <a
              class="link-cta"
              href="https://ottawamosque.ca/"
              rel="noopener noreferrer"
              target="_blank"
            >
              Website
            </a>
          </div>
        </div>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Masjid ar-Rahmah (AMA)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            South Nepean Muslim Community (SNMC)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Kanata Muslim Association (KMA)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Centre Islamique de l'Outaouais (CIO)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Jami Omar
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            MAC Barrhaven Islamic Centre (MAC BIC)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            MAC Qurtuba Islamic Centre (MAC QIC)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Masjid Bilal
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Centretown Mosque
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            The Ottawa Islamic Centre & Assalam Mosque
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Masjid Taqwa
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            East Ottawa Masjid and Community Centre (EOMCC)
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Bukhari Community Centre and Masjid
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Ottawa South Mosque
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Islam Care Centre
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Darul Uloom Ottawa
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
      <div
        class="rounded-lg border border-black/10 bg-white"
      >
        <button
          aria-expanded="false"
          class="w-full flex items-center justify-between px-4 py-3 text-left font-medium hover:bg-black/[0.03] transition-base"
        >
          <span>
            Ottawa Uyghur Centre
          </span>
          <svg
            aria-hidden="true"
            class="w-4 h-4 transition-transform "
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div>
      <div
        data-testid="stub"
        mosques="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
      />
    </div>
  </div>
</DocumentFragment>
`;
